.vib/apache/goss/vars.yaml

```diff
@@ -0,0 +1,8 @@
+service:
+  ports:
+    http: 80
+    https: 443
+hostAliases:
+  - ip: 127.0.0.1
+    hostnames:
+      - status.localhost
```

Comment:
Add these entries for `podSecurityContext` and `containerSecurityContext` as well:

```
podSecurityContext:
  fsGroup: 1002
containerSecurityContext:
  runAsUser: 1002
```